Projekty/Projekt1/Koziel_Krupinski_Lis/kamien_milowy_1.ipynb

```diff
@@ -0,0 +1,1572 @@
+{
```

Comment:
<p>Zakładając że dane są ułożone zgodnie z pewną naturalną kolejnością to tak (co jest tutaj po części prawdziwe). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>